physionet-django/notification/templates/notification/email/notify_credential_reference_response_reminder.html

```diff
@@ -0,0 +1,18 @@
+{% load i18n %}{% autoescape off %}{% filter wordwrap:70 %}
+Dear {{ applicant_name }},
+
+This is to remind you that your application for credentialed access to {{ SITE_NAME }} is still pending. We are currently waiting from your reference {{ application.reference_name }} to submit their response.
+
+We contacted your reference on {{ application.reference_contact_datetime }}
```

Comment:
Add full stop (`.`) at the end of this sentence.

Reply:
I just remembered the reason i didnot add full stop before.
It was because of the date, we got two fullstop, email sample attached for reference


```
From: localhost@localhost
To: amitupreti@fake_gmail.com
Date: Fri, 10 Mar 2023 17:22:24 -0000
Message-ID: <167846894445.58.2894547234442090599@1e454ba8beac>


Dear Amit Upreti,

This is to remind you that your application for credentialed access to
DataShare is still pending. We are currently waiting for your
reference Lorem Ipsum to submit their response.

We contacted your reference on Jan. 8, 2022, 1:49 p.m..

Please follow up with your reference so that they can submit their
response as soon as possible.

Please note that your application will be closed if we do not receive
a response from your reference within 15 days of the date of this
email.

It may take several days to process your request after we receive
response from your reference. Thank you for your understanding and
patience. You can follow the status of your application at:
http://localhost:8000/credential-application/.

Regards, The Development Team


```